src/webpack/loaders/devSiteEntry.js

```diff
@@ -0,0 +1,83 @@
+const lodashTemplate = require('lodash.template');
+const { getOptions } = require('loader-utils');
+
+const generateNavigationConfig = require('../loaderUtils/generateNavigationConfig');
+
+/**
+ * Generate the example with the supplied file.
+ * Don't use an arrow function
```

Comment:
Why not arrorw func? 

can we doc what template is?

Reply:
my comment got truncated on my frequent copying of the loader.

> Don't use an arrow function or you wont have access to `this`